kojiclient.py

```diff
@@ -2,6 +2,29 @@
 import json
 from utils import getScriptDir
 
+tag2branch = {
+	'fc20': 'f20',
+	'fc21': 'f21',
+	'fc22': 'f22',
+	'fc23': 'f23',
+	'fc24': 'f24',
+	'fc25': 'f25',
+	'el6': 'el6',
+	'el7': 'el7'
+}
+
+branch2tag = {
```

Comment:
Can you define ``branch2tag`` as a function? This way you can compute the mapping generically. Otherwise new mapping will have to be added for each new distribution manually.